### PR TITLE
Contrib update

### PR DESCRIPTION
**Pull request by sptramer**
Update contributor guidelines to clarify reviewer/maintainer responsibilities vs. where these are maintained by individual SIGs. Currently `sig-ui-ux` and `sig-release` have no such guidelines, so their links are empty and should be added when those resources are available.

### CODE REVIEW

**Comment by lmbr-pip** [comment]
Thanks for all the updates. I have a plan to move the contributing to O3DE pieces to the main README / CONTRIBUTING README to avoid duplicating content.

Reviewer and maintainer can then move to a standard guide.

We can work on shipping your changes first as they look great (will review formally later today)

File: community-membership.md [hunk 1]
```diff
@@ -6,39 +6,46 @@ The [O3DE community repo](https://github.com/o3de/community) contains informatio
...
 
-If a Maintainer fails to meet the requirements, they will be given a 1 month grace period to get back to the required number of Pull Requests per month.  After the 2nd month, if they haven't returned to the required number of reviews, they will be returned to Reviewer status.
+You must be nominated (self-nomination is allowed) for promotion from Reviewer to Maintainer. To do this, an issue should be created on the community repo for the supportive Maintainers to comment and add their support. Once you have the requisite support, the nomination Pull Request needs to be reviewed and approved during a SIG meeting.  The nominee should plan to attend and add it to the agenda for the SIG to review and, if appropriate, approve. Nomination Pull Requests that do not have the necessary support will be closed after 4 weeks of inactivity.
+
+## Becoming a Reviewer or Maintainer
+
+The SIGs maintain their own reviewer and maintainer standards, and have additional types of roles available in some cases. Maintainer standards are maintained in nomination issues, so each link goes to a "New Issue" form for nomination.
```
Comment (sptramer) [inline community-membership.md:37]:
```suggestion
The SIGs maintain their own reviewer and maintainer standards, and have additional types of roles available in some cases. These standards are maintained in nomination issues, so each link goes to a "New Issue" form for nomination.

```

File: community-membership.md [hunk 2]
```diff
@@ -4,41 +4,43 @@ Welcome to the Open 3D Engine! To learn more about contributing to the [O3DE cod
...
 
-If a Reviewer fails to meet the requirements, they will be given a 1 month grace period to get back to the required number of Pull Requests per month.  After the 2nd month, if they haven't returned to the required number of reviews, they will be returned to Contributor status.
+You must be nominated (self-nomination is allowed) for promotion from Contributor to Reviewer. To do this, an issue should be created on the SIG's repo for the supportive Reviewers and Maintainers to comment and add their support. For example, to nominate a reviewer on behalf of sig-content, an issue should be created in the [sig-content repository](https://github.com/o3de/sig-content/issues/new/choose).
 
-## How to become a maintainer
+Once you have the requisite support, the nomination Issue needs to be reviewed and approved during a SIG meeting. If possible, the nominee should attend and add it to the agenda for the SIG to review and, if appropriate, approve. Nomination Issues that do not have the necessary support will be closed after 4 weeks of inactivity.
```
Comment (lmbr-pip) [inline community-membership.md:21]:
```suggestion
Once you have the requisite support, the nomination issue needs to be reviewed and approved during a SIG meeting. If possible, the nominee should add the issue to the agenda for an upcoming SIG meeting and if possible attend the meeting. The SIG will then review and, if appropriate, approve. Nomination Issues that do not have the necessary support will be closed after 4 weeks of inactivity.

```

[the remark this answers is not in the document]
Comment (sptramer) [reply community-membership.md:21]: In this case we should, will add a link to the calendar and some text about joining meetings.

File: community-membership.md [hunk 3]
```diff
@@ -4,41 +4,43 @@ Welcome to the Open 3D Engine! To learn more about contributing to the [O3DE cod
 
 The [O3DE community repo](https://github.com/o3de/community) contains information about how to get started, how the community organizes, and more.
 
-To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/blob/development/CODE_OF_CONDUCT.md) first. 
+All contrubtors are required to follow our [Code of Conduct](https://github.com/o3de/o3de/blob/development/CODE_OF_CONDUCT.md).
 
-## How to become a reviewer
+## Contributing to Open 3D Engine
```
[the remark this answers is not in the document]
Comment (sptramer) [reply community-membership.md:9]: Going through and making some updates, it's becoming clear to me that not only is there duplicated information, it's fragmented out into lots of different spaces. This PR should mostly be considered a cleanup draft. I'll file an issue for followup.

File: community-membership.md [hunk 4]
```diff
@@ -4,41 +4,52 @@ Welcome to the Open 3D Engine! To learn more about contributing to the [O3DE cod
...
-To do this, an issue should be created on the community repo for the supportive Maintainers to comment and add their support. For example, to nominate a maintainer on behalf of sig-content, an Issue should be created in the [sig-content repository](https://github.com/o3de/sig-content/issues/new/choose).
 
-Once you have the requisite support, the nomination Issue needs to be reviewed and approved during a SIG meeting.  The nominee should plan to attend and add it to the agenda for the SIG to review and, if appropriate, approve.
-Nomination Issues that do not have the necessary support will be closed after 4 weeks of inactivity.
+## Maintainers
 
-Requirements to continue to be a Maintainer
-* 8+ Pull Requests reviewed per month
+A *Maintainer* role is an increased level of responsibility for a Reviewer. Maintainers aren't just responsible for performing review - they also have the ability to merge pull requests into the appropriate branch. Maintainers are high-quality Reviewers who take extra care when looking at a pull request, and are willing to trust the other Reviewers who approve work.
```
Comment (lmbr-pip) [inline community-membership.md:32]: IMHO we should call out maintainers have responsibility for the code they merge.

See changes in https://github.com/o3de/community/tree/main/sigs#contributors-reviewers-and-maintainers

**Review by lmbr-pip — APPROVED** [review]
Approving changes.

IMHO this page should be split as discussed in https://github.com/o3de/community/issues/159. 

This page should cover roles and expectations, the process of gaining those roles should be called out in owning areas documentation. ie how to become a reviewer or maintainer for a SIG should be in the SIG docs, how to be a chair should be in the SIG docs, TSC member in TSC docs.

General contributing, outside of defining the "contributor" role should be in the main CONTRIBUTING guide.

**Pull request merged**